src/web_app_skeleton/Procfile.dev.ecr

```diff
@@ -1,3 +1,4 @@
+system_check: script/system_check
```

Comment:
I think it’d be cool to have the system check never return SHELL so you can use it from other scripts or just run it on its own. Instead I think you could modify this line to `script/system_check && $SHELL` since shell is only necessary in the Procfile. That way the system check doesn’t have any conditional logic

Reply:
I like that better.